app/src/main/java/com/ikarabulut/Tests.java

```diff
@@ -0,0 +1,37 @@
+package com.ikarabulut;
+
+public class Tests {
```

Comment:
What if this class was called `Rules`?

That would also clean up the awkwardly named "TestsTest".

:)

Reply:
I clearly had TDD on my mind when I was making this class 😵‍💫

Reply:
Just made the change